openfold/data/data_pipeline.py

```diff
@@ -830,6 +831,15 @@ def read_template(start, size):
                     with open(path, "r") as fp:
                         hits = parsers.parse_hhr(fp.read())
                     all_hits[f] = hits
+                    fp.close()
+                    
+                elif (ext =='.sto') and (f.startswith("pdb")):
```

Comment:
Sorry my mistake. I saw in the test_data there is a pdb70_hits.hhr file and I thought the hmm results will also be named as such. I've changed the 2nd condition so that it check whether the file starts with 'hmm'